### PR TITLE
bump(*): get openshift/api scheduler changes

### PR DESCRIPTION
**Pull request by sjenning**
straight `hack/update-deps.sh` with patched glide https://github.com/Masterminds/glide/pull/1035
@soltysh @deads2k @mfojtik @ravisantoshgudimetla @derekwaynecarr

### CODE REVIEW

**Comment by deads2k** [comment]
/hold

contains k8s.io/* changes

**Comment by deads2k** [comment]
@sttts @damemi looks like component-base and kube-openapi

**Comment by sjenning** [comment]
@ericavonb can you verify openshift/service-serving-cert-signer bump is ok?

@adambkaplan @bparees can you verify openshift/source-to-image bump is ok?

**Comment by ravisantoshgudimetla** [comment]
/lgtm

@sjenning Thank you. Lgtm from scheduler side. We got the changes we need.

**Comment by mrogers950** [comment]
@sjenning the SSCS bump is ok because we have only just disabled the operator, not the parts that origin uses.

**Comment by sjenning** [comment]
flake https://github.com/openshift/origin/issues/22243

**Comment by enj** [comment]
@sjenning I am in the process of making a PR that removes SSCS from origin 4.0.

**Comment by soltysh** [comment]
https://github.com/openshift/origin/pull/22250 for cross reference

File: vendor/k8s.io/kube-openapi/pkg/handler/handler.go [hunk 1]
```diff
@@ -238,23 +238,6 @@ func jsonToYAMLValue(j interface{}) interface{} {
 			ret[i] = jsonToYAMLValue(j[i])
 		}
 		return ret
-	case float64:
```
Comment (soltysh) [inline vendor/k8s.io/kube-openapi/pkg/handler/handler.go]: This is wrong, we need to update `glide.yaml` to pick up the correct (ours) version of `kube-openapi`.

**Comment by soltysh** [comment]
Closing in favor of https://github.com/openshift/origin/pull/22267

**Pull request closed without merging**